sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -510,7 +513,7 @@ public String getId() {
      * @param options the request options
      * @return an {@link Mono} containing the  cosmos item resource response.
      */
-    public Mono<CosmosAsyncItemResponse> deleteItem(
+    public Mono<CosmosAsyncItemResponse<Object>> deleteItem(
```

Comment:
API change for @kushagraThapar and @moderakh to review.

Reply:
Same as above. Basically the same APIs.